pkg/entrypoint/entrypointer.go

```diff
@@ -104,6 +118,12 @@ func (e Entrypointer) Go() error {
 		_ = logger.Sync()
 	}()
 
+	// create a step path pipeline.StepPath/step-<step-name>/ if a step name is specified
+	// or pipeline.StepPath/step-unnamed-<step-index>/ if a step name is missing
+	// Create a symlink:
+	// ln -s pipeline.StepPath/<step-index> pipeline.StepPath/<step-name>
+	e.PostWriter.CreatePath(e.StepPath, e.StepPathLink)
+
```

Comment:
For the future, I think we should write in `/tekton/steps/by-index/<index>/…` and then link to elsewhere (either by name, or to keep backward compat, …)